src/server/simulator.js

```diff
@@ -167,11 +167,11 @@ Simulator.prototype.startSimulation = function () {
 
             this._state = Simulator.State.RUNNING;
         }.bind(this))
-        .fail(function (error) {
+        .catch(function (error) {
             log.warning('Error starting the simulation');
-            log.error(error);
```

Comment:
Could you please clarify why should we delete this line? Could it lead to loss of logging of error?

Reply:
We'd like to avoid double logs output. Error should be handled by caller